CONTRIBUTING.md

```diff
@@ -0,0 +1,66 @@
+# Code Style for Contributors
+
+* Use camelCase for functions and PascalCase for types.
+* Do not use tabs, indent code with 4 spaces.
+* If possible, a line of code should be at most 80 characters long.
+* Put a space between foreach, if, while etc. and the opening parenthesis.
+
+* Prefer to not include an empty set of parentheses in cases where there are only template parameters.
```

Comment:
On the preview of this file, there are looks to be extra space in-between some of the bullet points. Try removing the extra newline (7) and see if that helps. Same thing further down.